frontend/src/fixes/modal-layout.scss

```
@@ -0,0 +1,38 @@
+// Reproduce the platform modal internal layout for extension plugs rendered
+// inside modal iframes, where the host owns the outer dialog.
+//
```

Stylelint (17.12.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)